server/src/internalClusterTest/java/org/opensearch/action/admin/cluster/stats/ClusterStatsIT.java

```diff
@@ -505,6 +507,124 @@ public void testNodeRolesWithDataNodeLegacySettings() throws ExecutionException,
         assertEquals(expectedNodesRoles, Set.of(getNodeRoles(client, 0), getNodeRoles(client, 1)));
     }
 
+    public void testClusterStatsMetricFiltering() {
```

Comment:
Could we also parametrize the existing tests to turn on Transport level metrics filter ? You can mention all the metrics in the filter.

Reply:
The state of the cluster is retained within an IT, this will lead to conflicts on parametrizing within an IT.